### PR TITLE
docs(generators): fix generators links

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Docs

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
Yes
**Summary**

links were outdated and pointing to the master branch hence updated.

**Does this PR introduce a breaking change?**

No
**Other information**

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Don't think we need it, because next will be master in near future

File: packages/generators/README.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ const { addonGenerator, initGenerator, loaderGenerator, pluginGenerator } = requ
...
 
--   [**Plugin Generator**](https://github.com/webpack/webpack-cli/blob/master/packages/generators/plugin-generator.ts) : Creates a webpack plugin project, add starter plugin code and runs `webpack-defaults`
--   [**Loader Generator**](https://github.com/webpack/webpack-cli/blob/master/packages/generators/loader-generator.ts) : Creates a webpack loader project, add starter loader code and runs `webpack-defaults`
--   [**Init Generator**](https://github.com/webpack/webpack-cli/blob/master/packages/generators/init-generator.ts) : Generates new webapck configuration as per user requirements
--   [**Addon Generator**](https://github.com/webpack/webpack-cli/blob/master/packages/generators/addon-generator.ts) : Generates a webpack project conforming to `webpack-defaults`
+-   [**Plugin Generator**](https://github.com/webpack/webpack-cli/blob/next/packages/generators/src/plugin-generator.ts) : Creates a webpack plugin project, add starter plugin code and runs `webpack-defaults`
+-   [**Loader Generator**](https://github.com/webpack/webpack-cli/blob/next/packages/generators/src/loader-generator.ts) : Creates a webpack loader project, add starter loader code and runs `webpack-defaults`
```
Comment (snitin315) [inline packages/generators/README.md:30]: @evilebottnawi  these files moved inside the src folder as well.

**Comment by evilebottnawi** [comment]
No need it, it is like PR spam, we will fix doc urls after merge next with master

**Comment by snitin315** [comment]
Sounds Good.

**Pull request closed without merging**